spacecmd/src/spacecmd/utils.py

```diff
@@ -633,7 +634,7 @@ def parse_list_str(list_s, sep=","):
     >>> assert parse_list_str("a,,b,c") == ["a", "", "b", "c"]
     >>> assert parse_list_str("a:b:", ":") == ["a", "b", ""]
     """
-    return [p for p in list_s.split(sep)]
+    return [p for p in list_s.split(sep)] # pylint: disable=bad-option-value,unnecessary-comprehension
```

Comment:
Instead of disabling the pylint warning, this could be as simple as:
```
>>> def parse_list_str(list_s, sep=","):
...     return list_s.split(sep)
... 
>>> assert parse_list_str("") == [""]
>>> assert parse_list_str("a,b") == ["a", "b"]
>>> assert parse_list_str("a,b,") == ["a", "b", ""]
>>> assert parse_list_str("a,,b,c") == ["a", "", "b", "c"]
>>> assert parse_list_str("a:b:", ":") == ["a", "b", ""]

```

Reply:
Will modify it. I don't want to risk breaking code if I don't know what I am doing. So in those cases I rather disabled the warning for someone to revisit at a later stage.

Reply:
Fair enough, that's a good approach.

Reply:
Updated.